recipes/jigsaw/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "JIGSAW" %}
+{% set version = "0.9.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/dengwirda/jigsaw/archive/v{{ version}}.tar.gz
+  sha256: 75b2a7454b8fc3cc7d95475f915101d35ba0e434630f4668601772320f016ed2
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+
+test:
+  source_files:
+    - example.jig
+    - geo/bunny.msh
+    - out/temp.txt
+  commands:
+    - jigsaw example.jig
+
+about:
+  home: https://github.com/dengwirda/jigsaw
+  license_file: LICENSE.md
+  summary: |
+    JIGSAW is a Delaunay-based unstructured mesh generator for two- and
+    three-dimensional geometries.
+  description: |
+    JIGSAW is a computational library for unstructured mesh generation and
+    tessellation; designed to generate high-quality triangulations and
+    polyhedral decompositions of general planar, surface and volumetric
+    domains. JIGSAW includes refinement-based algorithms for the construction
+    of new meshes, optimisation-driven techniques for the improvement of
+    existing grids, as well as routines to assemble (restricted) Delaunay
+    tessellations and Voronoi complexes.
+
+    This package provides the underlying C++ source for JIGSAW; defining a
+    basic command-line interface and a C-format API. A MATLAB / OCTAVE based
+    scripting interface, including a range of additional facilities for file
+    I/O, mesh visualisation and post-processing operations can be found at
+    https://github.com/dengwirda/jigsaw-matlab.
+  doc_url: https://github.com/dengwirda/jigsaw
+  dev_url: https://github.com/dengwirda/jigsaw
+
+extra:
+  recipe-maintainers:
+    - xylar
+    - dengwirda
```

Comment:
@dengwirda, please confirm that you're willing to be a maintainer (which would entail merging automatically generated PRs when new versions of JIGSAW are released and troubleshooting to the best of your ability if the recipe needs to be updated).

Reply:
@xylar Yes, no problem -- happy to contribute here.